lib/vmdb/appliance.rb

```diff
@@ -78,6 +78,11 @@ def self.log_server_identity
         startup.info("Server Role: #{MiqServer.my_role}")
         s = MiqServer.my_server
         region = MiqRegion.my_region
+        rh_domain = MiqAeDomain.find_by(:name => "RedHat")
+        if rh_domain.version != rh_domain.available_version
+          startup.info("RedHat domain on disk differs from domain in db:")
+          startup.info("Current version - #{rh_domain.version}, Available version - #{rh_domain.available_version}")
+        end
```

Comment:
Maybe we can do this in `log_config` above, after we log the version, build, codename, ruby and rails version information.  Call a method to check the domain.